packages/beeq/src/components/date-picker/_storybook/bq-date-picker.stories.tsx

```diff
@@ -0,0 +1,288 @@
+    'validation-status': { control: 'select', options: [...INPUT_VALIDATION] },
+    type: { control: 'select', options: [...DATE_PICKER_TYPE] },
+    months: { control: 'number' },
+    value: { control: 'text' },
+    'format-options': { control: 'object' },
+    // Events
+    bqBlur: { action: 'bqBlur' },
+    bqChange: { action: 'bqChange' },
+    bqClear: { action: 'bqClear' },
+    bqFocus: { action: 'bqFocus' },
+    // Not part of the public API, so we don't want to expose it in the docs
+    noLabel: { control: 'boolean', table: { disable: true } },
+    hasLabelTooltip: { control: 'boolean', table: { disable: true } },
+    prefix: { control: 'boolean', table: { disable: true } },
+    suffix: { control: 'boolean', table: { disable: true } },
+    customDisallowedDate: { control: 'text' },
+  },
+  args: {
+    autofocus: false,
+    'clear-button-label': 'Clear value',
+    'disable-clear': false,
+    distance: 8,
+    disabled: false,
+    form: undefined,
+    min: undefined,
+    max: undefined,
+    locale: undefined,
+    'show-outside-days': false,
+    'first-day-of-week': 1,
+    name: 'bq-date-picker',
+    open: false,
+    'panel-height': 'auto',
+    placement: 'bottom-start',
+    placeholder: 'Enter your date',
+    skidding: 0,
+    strategy: 'absolute',
+    required: false,
+    'validation-status': 'none',
+    type: 'single',
+    months: 1,
+    value: undefined,
+    'format-options': {
+      day: 'numeric',
+      month: 'short',
+      year: 'numeric',
+    },
+    isDateDisallowed: undefined,
+    customDisallowedDate: undefined,
+  },
+};
+export default meta;
+
+type Story = StoryObj;
+
+const Template = (args: Args) => {
+  const tooltipTemplate = args.hasLabelTooltip
+    ? html`
+        <bq-tooltip class="ms-xs">
+          <bq-icon name="info" slot="trigger"></bq-icon>
+          You can provide more context detail by adding a tooltip to the label.
+        </bq-tooltip>
+      `
+    : nothing;
+  const labelTemplate = html`
+    <label class="flex flex-grow items-center" slot=${ifDefined(!args.optionalLabel ? 'label' : null)}>
+      Date picker label ${tooltipTemplate}
+    </label>
+  `;
+  const label = !args.optionalLabel
+    ? labelTemplate
+    : html`
+        <div slot="label" class="flex flex-1">
+          ${labelTemplate}
+          <span class="text-text-secondary">Optional</span>
+        </div>
+      `;
+  /**
+   * * Converts a Date object to an ISO 8601 string representation.
+   * This function is used only for demonstration purposes in Storybook.
+   */
+
+  const dateToIsoString = (date: Date): string => {
+    return date.toISOString().split('T')[0];
+  };
+
+  const isDate = (date: Date): boolean => {
+    if (!args.customDisallowedDate) {
+      return false;
+    }
+
+    const dateString = dateToIsoString(date);
+    return args.customDisallowedDate.split(',').some((date: string) => date.trim() === dateString);
+  };
+
+  return html`
+    <div class="w-[280px]">
```

Comment:
I would rather remove this and let the component take the whole width space available.